tests/tests/fee_tests.cpp

```diff
@@ -947,7 +1028,7 @@ BOOST_AUTO_TEST_CASE( stealth_fba_test )
 BOOST_AUTO_TEST_CASE( defaults_test )
 { try {
     fee_schedule schedule;
-    const limit_order_create_operation::fee_parameters_type default_order_fee;
+    const limit_order_create_operation::fee_parameters_type default_order_fee {};
```

Comment:
Why did you change this?

Reply:
We have added the braces because otherwise the tests project was not compiled with Clang on MacOs. The problem was in the fact, that If a program calls for the default initialization of a const object, the type of an object should have a user-provided default constructor. Since fee_parameters_type structure doesn't provide a default constructor, we have added an explicit initializer to compile the tests binary. So for now, we have kept this change.

Reply:
I wonder why there isn't an implicit default constructor since all variables in the struct have been assigned a default value.